domains/scolder.json

```
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "ScolderCreations",
+        "email": "ozor1xx@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "scolder"
```

Maybe you meant to use your `github.io` url?
```suggestion
        "CNAME": "https://scoldercreations.github.io/"
```

---

@TheBotlyNoob That's not a valid CNAME. It would be `scoldercreations.github.io`.

---

Sorry I got that wrong, I’ll fix it.